### DISCUSSION TITLE
Identified .Net upgrade and downloaded different version

### DISCUSSION THREAD

**Discussion by ja-law** [Romanitho/Winget-AutoUpdate#570] [General]
Hi, had a weird issue this morning when WAU ran on a machine. It identified an update for Microsoft Windows Desktop Runtime v7 but downloaded v6 , see WAU log entry below. Update failed as expected as machine is on latest versions of both .Net 6 & .Net 7. Ran winget upgrade manually and no update found for .Net 7 , as expected , so why did WAU think there was an update ?

10:26:46 - Checking application updates on Winget Repository...
-> Available update : Microsoft Windows Desktop Runtime - 7.0.16 (x64). Current version : > 6.0.27. Available version : 7.0.16.
10:26:50 - Updating Microsoft Windows Desktop Runtime - 7.0.16 (x64) from > 6.0.27 to 7.0.16...
10:26:56 - ##########   WINGET UPGRADE PROCESS STARTS FOR APPLICATION ID 'Microsoft.DotNet.DesktopRuntime.6'   ##########
10:26:56 - -> Running: Winget upgrade --id Microsoft.DotNet.DesktopRuntime.6 -e --accept-package-agreements --accept-source-agreements -s winget -h

Thanks

Jason

**Comment by Romanitho** [comment]
It's a winget thing. Had same behavior. I blacked list the app.

**Reply by ja-law** [reply]
Ok, good to hear someone else has seen this as well

**Comment by ja-law** [comment]
Known issue with winget
